pages/[sectionSlug]/[articleSlug].vue

```diff
@@ -181,11 +184,13 @@ const getGalleryLink = computed(() => {
           </div>
         </div>
         <hr class="black" />
-        <p class="type-label3 mt-2 mb-4">MORE {{ article.section.slug }}</p>
-        <article-recirculation
+        <p v-if="article?.section" class="type-label3 mt-2 mb-4">
+          MORE {{ article.section.slug }}
+        </p>
+        <!-- <article-recirculation
```

Comment:
did you mean to comment out the article-recirculation comp?

Reply:
no sorry - I was trying to fix that error and was eliminating things - I'll fix this!